ts/features/idpay/initiative/details/store/index.ts

```diff
@@ -38,17 +38,33 @@ const reducer = (
         details: pot.toError(state.details, action.payload)
       };
     // TIMELINE ACTIONS
-    case getType(idpayTimelineGet.request):
+    case getType(idpayTimelinePageGet.request):
+      if (action.payload.page === 0) {
+        return {
+          ...state,
+          timeline: pot.toLoading(pot.none)
+        };
+      }
       return {
         ...state,
         timeline: pot.toLoading(state.timeline)
       };
-    case getType(idpayTimelineGet.success):
+    case getType(idpayTimelinePageGet.success):
+      const currentTimeline = pot.getOrElse(state.timeline, {
+        lastUpdate: action.payload.lastUpdate,
+        operationList: []
+      });
       return {
         ...state,
-        timeline: pot.some(action.payload)
+        timeline: pot.some({
```

Comment:
I am not sure this is correct.

Can you think about what will happen if we request 2 times page 1?